docs/api/vast-tracker.md

```diff
@@ -744,7 +775,7 @@ vastTracker.on('overlayViewDuration', () => {
 vastTracker.overlayViewDuration();
 ```
 
-### verificationNotExecuted(macros)
+### verificationNotExecuted(macros) <a name="verificationNotExecuted"></a>
```

Comment:
I noticed that this method has actually 2 params 
`verificationNotExecuted(vendor, macros = {})`

Reply:
Nice catch!